composer.json

```diff
@@ -26,12 +25,22 @@
         "monolog/monolog": "^1.13.1",
         "smalot/pdfparser": "~0.9.24",
         "wallabag/tcpdf": "^6.2",
-        "true/punycode": "~2.0"
+        "true/punycode": "~2.0",
+        "psr/http-message": "^1.0",
+        "php-http/client-implementation": "^1.0",
+        "php-http/httplug": "^1.0",
+        "php-http/message-factory": "^1.0",
+        "php-http/discovery": "^1.0",
```

Comment:
I see that `j0k3r/safecurl` will no longer be used, I must analyse his behaviour to reimplement with  `httplug`